cmd/installer/pkg/install/install.go

```diff
@@ -240,9 +244,9 @@ func (i *Installer) Install(seq runtime.Sequence) (err error) {
 	if seq != runtime.SequenceUpgrade || !i.bootPartitionFound {
 		i.cmdline.Append("initrd", filepath.Join("/", i.Current, constants.InitramfsAsset))
 
-		syslinuxcfg := &syslinux.Cfg{
+		grubcfg := &grub.Cfg{
```

Comment:
Need to create this via the bootloader interface (method is still needed in the interface definition).